docs/source/library/python/data_isis.rst

```diff
@@ -0,0 +1,11 @@
+:mod:`IsisSpice` --- ISIS Attached SPICE Data Mix-in
```

Comment:
For all of these mix-in classes and the base class, it may be worth while to put them all under a single page. I'm not sure how that would look in the docs but it may reduce complexity of the docs overall.

Reply:
We could put IsisSpice and NaifSpice together, then IsisLabel and Pds3Label together? I think all 4 on one page is too much.

Reply:
I like the idea of grouping the way @jessemapel suggested: Spice sources on one page and Label type on another.

Reply:
I ended up restructuring the Python docs and splitting out the mix-ins and concrete drivers into their own directories.